crates/turbopack-ecmascript/src/references/mod.rs

```diff
@@ -725,6 +730,52 @@ pub(crate) async fn analyze_ecmascript_module(
                         )
                     }
 
+                    JsValue::WellKnownFunction(WellKnownFunctionKind::RequireContext) => {
+                        let args = linked_args(args).await?;
+                        if (1..=3).contains(&args.len()) {
```

Comment:
Why do we validate this again here if it's also done in `parse_require_context`?

Could we catch the error and show it with `span_err_with_code`?

Reply:
good point, fixed